scripts/reinstall-node-modules.js

```diff
@@ -0,0 +1,11 @@
+const {exec} = require('child_process');
+
+const isWindows = process.platform === 'win32';
+const command = `./tools/reinstall-node-modules${isWindows ? '.cmd' : ''}`;
+const executedBin = exec(command, error => {
```

Comment:
You need to pass `cwd` (should be zulip directory), and `encoding` to `utf8`  options to this.

Nit: `executedBin` should be named to something else like `proc`

Reply:
@cPhost I don't quite understand what do you mean about cwd and encoding.
I use __dirname variable which specifies the current direction name, I think it works just fine.
What about encoding?

Reply:
The `cwd` is null by default.  you need to passed it in as a option eg 
```javascript
exec(cmd, { cwd: path.resolve(__dirname, '../')}, ...)
```

For encoding it should be:
```javascript
executedBin.stdout.setEncoding('utf8')
executedBin.stderr.setEncoding('utf8')
```